independent-projects/tools/base-codestarts/src/main/resources/codestarts/quarkus/project/quarkus/base/src/main/resources/META-INF/resources/index.tpl.qute.html

```diff
@@ -4,27 +4,48 @@
VVxNFtDL1WCzOdD2C3C66CyAAiTvpcEozvoL6E3H/Vd3fL6fgwmO5ODNAl0SdH2hmi0j9wNpnP43g4jLrE5t24QMHa4xE6jFqlu1OmyWRntXManK4WFS1P2YBpYt3hMLO5zFl2sd/tjDyAZh/4yMn3dgBSlwuZu5xsxmMK2YagYnZslOTtIAVKK3BNAaRfx7WYzZ5ZpnsfQN/m5nILyzKmArcsBaPUNkxr28GOwvWYVX02BQmC0101bFaa82tzS1gIV37X7kdX58EAFZWoulhPWH2tlrPpdb7RWZ7DR5A1HZymz/xQClUlsfDMOnJ9wL8MnDtfEFW6rxuryvdI6fEoNAqkFjqoJw/xeZPRoIBS0Y4VbvCun6wjryhgPPGXdidUOOssdDueEed6QapvcwaIElPKfnaeJ3UXZsqPDez3YX+5KQAxH4WebGe7hv9uyV2C31Q+rE20cJt3z90AFkjAV1mS/3c8sBrm2AAUnUb0Za84J5HbrSO9Vvf2e4sbX3MfJbQWqJ5WUfT79HFWc4rk0LbIUpkm1yvTQ24uyJ+UbXH+wBsb7agm10TaaK1BM2kzY0ooxeAJIQrDvWgm3Hm0ou4KMEaxdgdU9pusZTTcP9XuoI+5DIfJoUNU0awxag6GgYm5a6LKuh3Qjw9KSalLxaM/cLQugp2UX63BpJcCfaGMXG2zXAw8BO5YVqsd/+YEOWU0qz7oaQ8YAROqyuXevl5TceuznmRgLVRt3RV6tMCcVZAar9ulvMcUClq8aWi3KfMpSr7/G5oNmCF8i29YkSBpZ2Q1YGLJo7cBsXow3Zl2WpxWKrj5IyqOstoQ46atZHlPXPtDZjkQzC9QsP3dtMxFq2CSQ/qY+rOeHqFM68m4hCMdfVr+J7b9Ib33ipmGGD0J+DeTsZq3W1WZP4//bSAWOA6QG78iP8Dazd1Mxk0C13JVrQUaHdgQcQCwq1b/6NG8HZnTR0LzVGEyGbY/QbPwg7D7U11XYEFLgCYQwIYIAcAuCMTGqQqV82IGSSFavJyU0EBrC6zw3LKDSTbmiPcmHpPDz6DVo3yww3ZmQVtyRswwcDQ7fiPMqwZa0WzxTY3EjmmixbxJUinTiPSaLt3Uh04c3EqMwSF+ZM4KPNw/AkC/LAzEooOEm0DLgTwlBS0TePl9n6Pbx1N2QpKAh51JEi44laRhnOtJOgKCTDLAjsFJJqiITNphY8xKsjAyyaQ9hufopANiMi/piMhcnHRCeN5POhMBzS3pgtAKTlr5lCJ5Bb8qklex6DufcA2TfuB+Pmlg1zGx9AxKGakpqZhQgvFXRUpepsmLmQw8UoeSS8wLA8/SF/XmSlLqi0Jm5qpcGnNieCU8ckPSLBAokJI6izXNEjxSetoEZq4ftmnIKe6pOiaFAlk7r/DiOY0QwHPSYSUzzXSjEDz8xXCYRJk8MTCGczHPGLTtq1WWTx6gH65GKHFa3Px1pXrDxcRNpigmG6W2cGQ+ip6iM1EPQRFW6BQvZSlJSGMTqMRNASLIS3UjKZ1VTMQXibnMUMZlBqyiffDCQabvuT6XpUJLlOQmUskTyhjzc5YWleLINdGVPC/h4r5IggqSEsltuBeVGaYsb2johSxedPGGJDo6XK7zAqtyfWjPxBhpHGGDetKsitQPOzUnkzCGYBsYgoMsNgjJWVZqdKTjSXvk5qjJNKNnet68U65kac+faNXuNDYa3XrMUBCs6POwbcu5wySLrIah8yRD4xU/2yADLQOh7CPOkaDlcxOzlA6INDz7JE8A1yA8m7mytTtTgjTGzXexZwo5yTObdiT5CVCMZ9wTRSgZ8sqTCCLgIUsl1NjXIkjwfG1JztBXt96a6QXM1IwXNQ0S07P8/0sE+3n6gSAJGkEnGH/3IQALr73H5wgnOMMFrnDjzoMnL95/meBLMwDbQBzf88NF8RdgkMGGGGqY4UYYaZTRxuAJxBckWIj8/8GMM94EEwmECRchUpRoQjFixYmXIFGSZClSpUmXIVOWbDly5Zlksimmmma6GURmmmW2OcTBxAYNGp202GdNurRbZZuNYYc2T9VbECzCHp2WaHHey3DAatv95le/W2+XKy7bTUKqm8w1clZX3XLdDTd9oXDPbXf0UPrRPA/d94DKN31azaWWT0tDZy29AgZGhcxMihT7qkSZUuUqVThinWpVatTq1e+YR/bY67jHXngSjoQT4Uy4EK6EW7iHR3iGV3iHT3wXvvbZ75DDLjjgoIua7YgBTjkd7BioI0xRIBg9EYJeUqWpKFQjtnFkCGVRqssJnzKkMIjvWbyOMMKYYgdH7mYwEAbSQBkRBtqINDBGlBEdxw8iwvMI678bs1Eu1an945CUAE1R/tsZcmVFcYeiEBwLBtJA3ZMI3L94VfGoyNBHpVG6/0/JPoVu3cG4olm0qsINlHaBlIJJTP6kNHNu5s1m7piS06tQ9z1UfTHKnoxiOCn50BeSDmlkd1pT2qmsFpWk1ZviNiRqIyHCxihBLZuZdSijVsGvMMSr+kivUsUtjckpFLvQmqwihZnXwsh5GpupmUpJyUxOFD1JU5ISHLTYpEYqMUSTEipy2JdCCvogBikEdhCicMhmglcI66Ui3hNUHBhSYj2KifGYFe2RBgb3i+xQEagQQoQHQQL8eVF/J5CbMAOFQOAwHDBb92eP2B0B8XWDQHzc9GFviAovCA56ebg6C6+uHhCwy/92Gef/ASHdIAInyhkj8C5idsF0w4o/uN8QAeSqiFPL37wT8nC+/qzcX79XnlV71SV/vzeBbNLoQp6uPykP1x+U2SVTXlQn1UH1v5kwBOF98978AxMwMxitPyp31++UR9VOdUnvdyYkywzzL+/jH/yr6/tRpUbTI4hO3gCHne0t8CviCvJ6p6EsbiQh/qq++/kT5jdH+KsnvzB4u8EUqJhUTM5CzY/NCrNi4h8z9nAJxBJ005kthDklKzud9Y8gwyaI2Hn7a3YR9exe1AAA) format('woff2'),
```

Comment:
Thats a big chunk of blob to have at beginning of file - will it make a difference for the browswer if we put this last so actual content of page is what we show user first ?

Reply:
The CSS has to be in the head of the file for it to work. One of the parameters set by erin was that everything had to be in one file (images, fonts, etc) so there is no outside dependencies. The large blogs are Base64 encoded images (header background) and svg (logo) as well as the Open Sans font.

Reply:
y. people aren't going to read the raw source of this file, Max. And if they do, it's pretty obvious what this is.. I don't have an issue here.